atom/browser/api/atom_api_web_contents.cc

```diff
@@ -893,21 +939,12 @@ void WebContents::DidFinishNavigation(
 void WebContents::SecurityStyleChanged(
     content::SecurityStyle security_style,
     const content::SecurityStyleExplanations& explanations) {
-    std::string type = "unknown";
-    switch (security_style) {
-      case content::SECURITY_STYLE_UNAUTHENTICATED:
-      case content::SECURITY_STYLE_AUTHENTICATION_BROKEN:
-        type = "insecure";
-        break;
-      case content::SECURITY_STYLE_WARNING:
-        type = "warning";
-        break;
-      case content::SECURITY_STYLE_AUTHENTICATED:
-        type = "secure";
-        break;
-      default: break;
+    if (explanations.displayed_insecure_content &&
+        security_style == content::SECURITY_STYLE_UNAUTHENTICATED) {
+      Emit("security-style-changed", "passive-mixed-content");
```

Comment:
I really think we're passing through the wrong info here. `passive-mixed-content` is additional information about the security style and it should be a 3rd param imo because it's somewhat independent of the security style